src/walberla_bridge/tests/EKinWalberlaImpl_unit_tests.cpp

```diff
@@ -74,7 +74,11 @@ BOOST_DATA_TEST_CASE(dimensions, bdata::make(all_eks()), ek_generator) {
 }
 
 BOOST_AUTO_TEST_CASE(stencil_size) {
-  auto constexpr stencil_size = std::size_t{9u};
+  // Does it makes sense to check  for a stencil of 9 (D3Q19) as advection
+  // needs all 27 direction to operate properly anyway. I we want it we need
+  // to add the flux accesses for these different FluxCounts
+  // auto constexpr stencil_size = std::size_t{9u};
+  auto constexpr stencil_size = std::size_t{13u};
```

Comment:
The number you put here is not physically inspired, it's a unit test for the stencil size. The discretization (and the codegen) should work for any number of fluxes, only the error order changes."
```suggestion
  auto constexpr stencil_size = std::size_t{13u};
```